devops/git/workflow.md

```diff
@@ -9,7 +9,8 @@ after the initial commit, with the correct protection rules in place.
 
 * `master` should be the working development branch [UNSTABLE]
 * `feature/*` branches created from master for each feature
-* `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly.
+* `fix/*` branches are for non-urgent fixes, and follow the same flow as feature branches.
+* `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly for urgent fixes only. However, they can be [cherry picked](https://git-scm.com/docs/git-cherry-pick) into other branches if absolutely necessary.
```

Comment:
Could we possibly add as @AlexCatch Suggested to merge backwards once the pr has been approved into staging and prod.
```suggestion
 * `hotfix/*` branches should only be created off of `staging` or `production` and merged accordingly for urgent fixes only. However, they can be [cherry picked](https://git-scm.com/docs/git-cherry-pick) into other branches if absolutely necessary. Once merge has been successful, this change needs to be pushed down the chain to ensure that master has all the changes that are on staging, and staging has all the changes that are on prod.
```

Reply:
@dextermb, I have slain the magical space. one more and I can level up.